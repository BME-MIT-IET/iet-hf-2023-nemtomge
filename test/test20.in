loadmap map.json 
start V1 
set Scientist V1 PL1 
give Material V1 aminoacid 100 
give Material V1 nucleotide 100 
craft V1
0
has Crafted V1 Stun 
has Material V1 aminoacid 
has Material V1 nucleotide
exit
