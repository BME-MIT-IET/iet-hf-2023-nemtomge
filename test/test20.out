Map from file map.json loaded
The game has been started. Players are: V1 
Scientist V1 set on field PL1 
Scientist V1 has been given material aminoacid quantity: 100 
Scientist V1 has been given material nucleotide quantity: 100 
Choose one to craft: 
0 - Cancel
Scientist V1 doesn't have crafted agent Stun
Scientist V1 has material aminoacid quantity: 55
Scientist V1 has material nucleotide quantity: 55
